api/database/seeders/UatSeeder.php

```diff
@@ -17,7 +17,6 @@ public function run()
         $this->call(ClassificationSeeder::class);
         $this->call(DepartmentSeeder::class);
         $this->call(TeamSeeder::class);
```

Comment:
I like this change.  It might be worth adding a link to your new doc page here.  Something like "To manually promote a Sign In Canada user to admin, see....".  Perhaps the same comment in the UserSeederLocal, too.

Reply:
Updated in a325b3978736ca574eea3563f0e0091c3d88ee83.